packages/scene-composer/src/components/panels/scene-components/AnchorComponentEditor.tsx

```diff
@@ -118,10 +123,14 @@ export const AnchorComponentEditor: React.FC<IAnchorComponentEditorProps> = ({
   );
 
   const iconOptions = useMemo(() => {
-    return Object.keys(SCENE_ICONS).map((sceneIcon) => ({
-      label: intl.formatMessage(i18nSceneIconsKeysStrings[sceneIcon]) || sceneIcon,
-      value: sceneIcon,
-    }));
+    return Object.keys(SCENE_ICONS)
+      .filter((icon) => (!tagStyle && icon !== 'Custom') || tagStyle)
+      .map((sceneIcon) => {
+        return {
+          label: intl.formatMessage(i18nSceneIconsKeysStrings[sceneIcon]) || sceneIcon,
+          value: sceneIcon,
+        };
+      });
```

Comment:
you should need to do this same thing in https://github.com/awslabs/iot-app-kit/blob/main/packages/scene-composer/src/components/panels/scene-rule-components/SceneRuleTargetIconEditor.tsx#L22-L25 otherwise the custom icon option will show up there

Reply:
fixed